magenta-lib/src/main/scala/magenta/Lookup.scala

```diff
@@ -73,4 +74,6 @@ case class DeployInfoLookupShim(deployInfo: DeployInfo, secretProvider: SecretPr
   }
 
   def stages = deployInfo.knownHostStages
+
+  def getLatestAmi(region: String)(tags: Map[String, String]): Option[String] = ???
```

Comment:
Why isn't this implemented?

related: Good to use the override keyword if you're overriding because you get the compile-time check that it actually overrides something.